3_javascript/javascript1/week1/script.js

```diff
@@ -25,4 +25,50 @@ if (shouldShowResultInDogYears) {
 }
 
 
+// Housey pricey (A house price estimator)
+let widthPeter = 8;
+let deepPeter = 10;
+let heightPeter = 10;
+let gardenSizePeter = 100;
+let costPeter = 2500000;
 
+let widthJulia = 5;
+let deepJulia = 11;
+let heightJulia = 8;
+let gardenSizeJulia = 70;
+let costJulia = 1000000;
+
+function housePriceCalculator(width, deep, height, gardenSize) {
+    let volumeInMeters = width * deep * height;
+    let housePrice = volumeInMeters * 2.5 * 1000 + gardenSize * 300;
+    return housePrice;
+}
+
+let realPricePeter = housePriceCalculator(widthPeter, deepPeter, heightPeter, gardenSizePeter);
+let realPriceJulia = housePriceCalculator(widthJulia, deepJulia, heightJulia, gardenSizeJulia);
+
+if (realPricePeter < costPeter) {
+    console.log("Peter is paying too much.");
+} else {
+    console.log("Peter is paying just right or less.");
+}
+
+if (realPriceJulia < costJulia) {
+    console.log("Julia is paying too much.");
+} else {
+    console.log("Julia is paying just right or less.");
+}
```

Comment:
Perfect 💯  solution!
One small suggestion, if you use `const` instead of `let`, as you can see value of the variables don't change. So, it's better to opt for `const`. This will help you to avoid accidental reassignment of these variables and also your code will be more readable.

Reply:
Thank you for advice 👍